code/game/objects/items.dm

```diff
@@ -1196,7 +1196,7 @@
 	add_filter(HOVER_OUTLINE_FILTER, 1, list("type" = "outline", "size" = 1, "color" = outline_color))
 
 /// Called when a mob tries to use the item as a tool. Handles most checks.
-/obj/item/proc/use_tool(atom/target, mob/living/user, delay, amount=0, volume=0, datum/callback/extra_checks)
+/obj/item/proc/use_tool(atom/target, mob/living/user, delay, amount=0, volume=0, datum/callback/extra_checks, interaction_key)
```

Comment:
You don't seem to have actually made `interaction_key` used in the proc if I'm not mistaken

Reply:
Yeah I was testing it just now and realized it wasn't working, I think it mightve gotten shot while i updated it from 8 months ago, but I think I'm gonna scrap it anyways because it makes it infinitely slower.